language/de-DE/joomla.ini

```
@@ -176,30 +182,32 @@ JFIELD_META_RIGHTS_DESC="Beschreibung, welche Rechte andere Benutzer dieser Seit
 JFIELD_META_RIGHTS_LABEL="Inhaltsrechte"
 JFIELD_ORDERING_DESC="Reihenfolge für Beiträge in dieser Kategorie"
 JFIELD_ORDERING_LABEL="Reihenfolge"
+JFIELD_PASSWORD_INDICATE_INCOMPLETE="Das Passwort entspricht nicht den Anforderungen."
+JFIELD_PASSWORD_INDICATE_COMPLETE="Passwort angenommen."
 JFIELD_PUBLISHED_DESC="Veröffentlichungstatus setzen"
 JFIELD_TITLE_DESC="Beitragstitel"
 
 JGLOBAL_ADD_CUSTOM_CATEGORY="Neue Kategorie hinzufügen"
 JGLOBAL_ARTICLES="Beiträge"
-JGLOBAL_CONFIRM_DELETE="Sicher, dass gelöscht werden soll? Durch die Bestätigung werden die ausgewählten Einträge dauerhaft gelöscht!"
+JGLOBAL_CONFIRM_DELETE="Sicher, dass gelöscht werden soll? Durch die Bestätigung werden die ausgewählten Einträge dauerhaft gelöscht."
 JGLOBAL_FIELDS="Felder"
 JGLOBAL_AUTH_ACCESS_DENIED="Zugriff verweigert"
 JGLOBAL_AUTH_ACCESS_GRANTED="Zugriff erlaubt"
-JGLOBAL_AUTH_BIND_FAILED="Die Verbindung zum LDAP-Server ist <strong>fehlgeschlagen</strong>"
+JGLOBAL_AUTH_BIND_FAILED="Die Verbindung zum LDAP-Server ist <strong>fehlgeschlagen</strong>."
 JGLOBAL_AUTH_CANCEL="Authentifizierung abgebrochen"
 JGLOBAL_AUTH_CURL_NOT_INSTALLED="Curl ist nicht installiert"
-JGLOBAL_AUTH_EMPTY_PASS_NOT_ALLOWED="Ein leeres Passwort ist nicht zulässig"
+JGLOBAL_AUTH_EMPTY_PASS_NOT_ALLOWED="Ein leeres Passwort ist nicht zulässig."
 JGLOBAL_AUTH_FAIL="Authentifizierung <strong>fehlgeschlagen</strong>"
 JGLOBAL_AUTH_FAILED="Die Authentifikation von %s ist fehlgeschlagen"
 JGLOBAL_AUTH_INCORRECT="Benutzername/Passwort ist falsch"
-JGLOBAL_AUTH_INVALID_PASS="Benutzername und Passwort falsch oder das Benutzerkonto existiert noch nicht!"
+JGLOBAL_AUTH_INVALID_PASS="Benutzername und Passwort falsch oder das Benutzerkonto existiert noch nicht."
 JGLOBAL_AUTH_INVALID_SECRETKEY="Der Sicherheitscode der Zwei-Faktor-Authentifizierung ist ungültig."
 JGLOBAL_AUTH_NO_REDIRECT="Die Umleitung zum Server %s ist nicht möglich"
-JGLOBAL_AUTH_NO_USER="Benutzername und Passwort falsch oder das Benutzerkonto existiert noch nicht!"
+JGLOBAL_AUTH_NO_USER="Benutzername und Passwort falsch oder das Benutzerkonto existiert noch nicht."
 JGLOBAL_AUTH_NOT_CONNECT="Es konnte keine Verbindung zum Authentifizierungsdienst hergestellt werden."
-JGLOBAL_AUTH_NOT_CREATE_DIR="Das Speicherverzeichnis „%s“ kann nicht erzeugt werden. Bitte die tatsächlichen Dateirechte überprüfen."
+JGLOBAL_AUTH_NOT_CREATE_DIR="Das Speicherverzeichnis \"%s\" kann nicht erzeugt werden. Bitte die tatsächlichen Dateirechte überprüfen."
```

```suggestion
JGLOBAL_AUTH_NOT_CREATE_DIR="Das Speicherverzeichnis „%s“ kann nicht erzeugt werden. Bitte die tatsächlichen Dateirechte überprüfen."
```
Bitte vorerst so lassen.